src/libs/actions/Report.js

```diff
@@ -329,6 +329,12 @@ function addActions(reportID, text = '', file) {
         isLastMessageDeletedParentAction: null,
     };
 
+    const report = ReportUtils.getReport(reportID);
+
+    if (lodashGet(report, 'notificationPreference', null) === CONST.REPORT.NOTIFICATION_PREFERENCE.HIDDEN) {
```

Comment:
Should we combine this to a new `ReportUtils.getNotificationPreference` helper?

Reply:
Added - 8907434.
Implemented - 5699f67.